public/components/getting_started/components/getting_started_integrationCards.tsx

```diff
@@ -148,7 +149,11 @@ export const IntegrationCards = () => {
               description={integration.description}
               data-test-subj={`integration_card_${integration.name.toLowerCase()}`}
               titleElement="span"
-              href={`/app/integrations#/available/${integration.name}`}
+              onClick={() =>
+                coreRefs?.application?.navigateToApp(observabilityIntegrationsID, {
+                  path: `#/available/${integration.name}`,
+                })
```

Comment:
using onclick user won't be able to open link in new tab, can this use http.basePath.get()? but not a big issue since it's getting started page

Reply:
UX perspective is to load integrations in-place in the same tab.